packages/backend/src/libp2p/connectionsManager.tor.test.ts

```diff
@@ -19,6 +30,15 @@ beforeEach(() => {
   connectionsManager = null
 })
 
+afterEach(async () => {
+  if (connectionsManager) {
+    await connectionsManager.closeAllServices()
+    if (connectionsManager.libp2pInstance) {
+      await connectionsManager.libp2pInstance.stop()
+    }
+  }
+})
+
```

Comment:
Shouldn't stoping libp2pinstance be moved to closeAllServices?

Reply:
Maybe, I wasn't sure about that because theoretically IPFS stops libp2p however I see that there is a problem with that in tests. That's why I added additional stop here.

Reply:
and does it work?

Reply:
I think we should include that in closeAllServices same as the rest of services

Reply:
Moved to closeAllServices